jinahub/indexers/storage/MongoDBStorage/__init__.py

```diff
@@ -0,0 +1,123 @@
+__copyright__ = "Copyright (c) 2021 Jina AI Limited. All rights reserved."
```

Comment:
Ah one more thing: can you extract this file to `mongodb_storage.py`, and put in `__init__.py`
```python
from mongodb_storage import  MongoDBStorage
```